src/pages/documentation/mobile-core/identity/event-reference.md

```diff
@@ -1,293 +0,0 @@
-# Identity event reference
```

Comment:
is this doc already migrated to the corresponding github repo?

Reply:
No, we haven't started to migrate the event reference docs to the GitHub repos for core extensions. @praveek do we have any plan for this?

Reply:
ok, if you have a task for it maybe link it to this so you can reference these files

Reply:
I have moved the event reference pages for core extensions to the Github repo in this PR: https://github.com/adobe/aepsdk-core-android/pull/396